pkg/cmdlets/auth/login.go

```diff
@@ -58,8 +58,8 @@ func AuthenticateWithInput(
 			if !uniqueUsername(credentials) {
 				return errs.Wrap(err, "uniqueUsername failed")
 			}
-			if err := promptSignup(credentials, out, prompt, auth); err != nil {
-				return errs.Wrap(err, "promptSignup failed")
+			if err := SignupWithBrowser(out, auth, prompt); err != nil {
+				return errs.Wrap(err, "SignupWithBrowser failed")
```

Comment:
Should just error out if we reach this. Launching the browser without context while in a prompt flow will just cause confusion.